PeerioTechnologies/peerio-desktop
# style: remove 'all icons color' state from AppNav menu

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/13223/desktop-avatar-menu-hover-states
 
#### Testing instructions  
*    remove the initial full-colour state from avatar menu items
*    all items should be grey until hovered. Current hover states should be maintained
*    active tabs should still be reflected in the menu when setting page is open

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
